src/pages/guides/authentication/ServerToServerAuthentication/IMS.md

```diff
@@ -15,15 +15,15 @@ Parameters can be sent in the body or as query parameters. Passing parameters in
 |`grant_type`|Yes|Value should always be `client_credentials`|
 |`client_id`|Yes|The Client ID obtained from the [Adobe Developer Console](/console)|
 |`client_secret`|Yes|The value of client secret obtained from the [Adobe Developer Console](/console)|
-|`scopes`|Yes|The list of comma separated scopes you are requesting. To see the scopes available to your credential, visit your project your the [Adobe Developer Console](/console)|
+|`scope`|Yes|The list of comma separated scopes you are requesting. To see the scopes available to your credential, visit your project your the [Adobe Developer Console](/console)|
```

Comment:
The end of the sentence should also be fixed : "visit your project your the ..."